setup.py

```diff
@@ -105,7 +105,7 @@
     "amd": "optimum-amd",
     "quanto": ["optimum-quanto>=0.2.4"],
     "dev": TESTS_REQUIRE + QUALITY_REQUIRE,
-    "tests": TESTS_REQUIRE,
+    "tests": TESTS_REQUIRE + ["onnxslim>=0.1.53"],
```

Comment:
here you can add it directly to the TESTS_REQUIRE list

Reply:
OK